app/assets/javascripts/components/components/load_more.jsx

```diff
@@ -1,9 +1,9 @@
 import { FormattedMessage } from 'react-intl';
 
 const LoadMore = ({ onClick }) => (
-  <a href='#' className='load-more' onClick={onClick}>
+  <span tabIndex='0' className='load-more' role='button' onClick={onClick}>
```

Comment:
You can just put `role="button"` on the `<a href>`. It's better than turning it into a span.

We're working on some big updates to `eslint-plugin-jsx-a11y` that should make it more precise and unambiguous: https://github.com/evcohen/eslint-plugin-jsx-a11y/pull/194